app/controllers/v3/service_route_bindings_controller.rb

```diff
@@ -248,6 +251,12 @@ def route_binding_not_found!
     resource_not_found!(:service_route_binding)
   end
 
+  def route_binding_not_found_with_message!(service_route_binding)
+    type = service_route_binding.is_a?(RouteBinding) ? 'route' : 'binding'
+    state = service_route_binding.last_operation.state
+    resource_not_found_with_message!("Creation of binding #{type} #{state}")
```

Comment:
Similar to the service credential bindings controller, this error should be returned for "Creation" and "Deletion", thus the `last_operation.type` needs to be taken into account.